tensorflow/lite/tools/cmake/modules/flatbuffers.cmake

```diff
@@ -42,3 +42,18 @@ add_subdirectory(
   EXCLUDE_FROM_ALL
 )
 remove_definitions(-DNOMINMAX)
+
+# The host-side flatc binary
+include(ExternalProject)
+
+ExternalProject_Add(flatbuffers-flatc
```

Comment:
Oh I'm looking at this too. Did you verify it worked well with cross compilation?

Reply:
@terryheo 
> Oh I'm looking at this too. Did you verify it worked well with cross compilation?

Yes, I use the cross-tool for tflite and the host-tool for flatc.
I could see the mutable interface was generated successfully.
```
/tensorflow/tensorflow/lite/tools/cmake/build$ ls kernels/schema/tensorflow/lite/schema/mutable/schema_generated.h
kernels/schema/tensorflow/lite/schema/mutable/schema_generated.h
```